.github/workflows/tag-service.yml

```diff
@@ -0,0 +1,57 @@
+name: Tag Service
+
+on:
+  workflow_dispatch:
```

Comment:
Who is authorized to dispatch this workflow?

Reply:
I'll make a new `release-managers` group that can dispatch it.